scorecard/profile_data/indicators/repairs_maintenance_spending.py

```diff
@@ -48,6 +49,35 @@ class RepairsMaintenanceSpending(SeriesIndicator):
             "100",
         ],
     }
+    formula_v2 = {
+        "text": "= (Repairs and maintenance expenditure / (Property, Plant and Equipment + Investment Property)) * 100",
+        "actual": [
+            "=", 
+            "(",
+            {
+                "cube": "capital_v2",
+                "item_codes": ["4100"],
```

Comment:
there's no code to highlight - we're summing up all item codes available (filtered to only repmaint capital spending)

https://github.com/OpenUpSA/municipal-data/blob/master/scorecard/profile_data/indicators/repairs_maintenance_spending.py#L112
https://github.com/OpenUpSA/municipal-data/blob/master/scorecard/profile_data/api_data.py#L315

Does an empty item_codes list do the trick in terms of presenting the formula?

Reply:
or something like this
![Uploading Screenshot_2022-04-13_17-06-56.png…]()

Reply:
Added a new parameter that can set a item description such as "Total"